api-schema/src/main/java/org/stellar/anchor/api/callback/PutCustomerRequest.java

```diff
@@ -113,10 +113,10 @@ public class PutCustomerRequest {
   String idCountryCode;
 
   @SerializedName("id_issue_date")
-  Instant idIssueDate;
+  String idIssueDate;
```

Comment:
I think there is a reason that we use `Instant`. Can you explain a little the reason we want to use `String` instead?

Reply:
Oops, this is a mistake, it should be an `Instant`.